extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/runtime/MongoOperations.java

```diff
@@ -195,6 +196,7 @@ public MongoCollection mongoCollection(Class<?> entityClass) {
         return database.getCollection(entityClass.getSimpleName(), entityClass);
     }
 
+    @NotNull
```

Comment:
We didn't use this anotation anywhere else so can you give some details of why you add it here ?

Reply:
That looks like an experiment that wasn't cleaned up.  I'll do so now.